docs/core/quickstart-from-source.md

```diff
@@ -13,10 +13,8 @@ Now, before you proceed install git and jdk8 or jdk11, then do:
 ```bash
 git clone git://github.com/graphhopper/graphhopper.git
 cd graphhopper; git checkout master
-# install npm, can be omitted if no UI is needed
-wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash && \. $HOME/.nvm/nvm.sh && nvm install node
-# create main.js via npm, can be omitted if no UI is needed
-cd web && npm install && npm run bundleProduction && cd ..
+# fetches main.js, can be omitted if no UI is needed
+cd web/src/main/resources/ && ZFILE=/tmp/gh.jar && wget -O $ZFILE "https://repository.sonatype.org/service/local/artifact/maven/redirect?r=central-proxy&g=com.graphhopper&a=graphhopper-web&v=LATEST" && unzip $ZFILE assets/js/main.js && rm $ZFILE && cd ../../../..
```

Comment:
With this simple trick we fetch the latest main.js that is bundled in the maven release and there is no longer a need to install npm if no JS development is done. I think with this it is easy enough and ready to merge - what do you think @boldtrn ?